src/styles/blocks/form.scss

```diff
@@ -0,0 +1,43 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  margin-bottom: 48px;
+
+  @include on-tablet {
+    margin-bottom: 80px;
+  }
+
+  &__input {
+    padding: 14px 16px;
+    background-color: $input-bg-color;
+    border: none;
+    border-radius: 8px;
+    outline: none;
+    box-shadow: none;
+
+    font-family: $main-font;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 19.6px;
+
+    &::placeholder {
+      color: $input-placeholder-color;
+    }
+
+    &:hover {
+      border: 1px solid $input-hover-border-color;
+      color: $main-color;
+    }
+
+    &:active,
+    &:focus {
+      border: 1px solid $input-active-border-color;
```

Comment:
make it transparent by default, and when hovering - change the color.
This will help prevent form jumps